src/main/kotlin/com/linktime/plugin/jira/insightobjectgraph/impl/objects/ListObjectsServiceImpl.kt

```diff
@@ -109,6 +109,31 @@ constructor(private val insightApiFacade: InsightApiFacade,
                 .let { ListObjectsService.InsightObjectListRootNode(it) }
     }
 
+    override fun getObjectsWithIql(objectTypeName: String,
```

Comment:
This method contains duplicated code with "getObjects". A separate method does not seem necessary if the iqlString was defined as nullable (as is done with schemaId).

Reply:
Refactored to have only getObjects with the additional parameter